mmtrack/models/trackers/oc_sort.py

```diff
@@ -0,0 +1,478 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import lap
+import numpy as np
+import torch
+from addict import Dict
+from mmcv.runner import force_fp32
+from mmdet.core import bbox_overlaps
+
+from mmtrack.core.bbox import bbox_cxcyah_to_xyxy, bbox_xyxy_to_cxcyah
+from mmtrack.models import TRACKERS
+from .base_tracker import BaseTracker
+
+
+@TRACKERS.register_module()
+class OCSORT_Tracker(BaseTracker):
```

Comment:
`OCSORTTracker` to be consistent with the naming conventions? Also the file name -> `ocsort_tracker.py`.

Reply:
Have renamed them to keep consistent with the naming convention.